commands/post.go

```diff
@@ -125,11 +132,25 @@ func printPost(c client.Client, post *model.Post, usernames map[string]string, s
 		}
 	}
 
+	postTime := time.UnixMilli(post.CreateAt).UTC()
+	createdAt = postTime.Format(PostTimeFormat)
+
 	if showIds {
-		printer.PrintT(fmt.Sprintf("\u001b[31m%s\u001b[0m \u001b[34;1m[%s]\u001b[0m {{.Message}}", post.Id, username), post)
```

Comment:
I'm thinking we only want to print the timestamp if the user selected the since option, which would require altering `printPost` to take take in a boolean of whether that option was selected.

Reply:
Updated `printPost` to print timestamp only with **--since** flag